src/client/pages/RetroCertsAuthPage/index.js

```diff
@@ -47,43 +51,71 @@ function RetroCertsAuthPage() {
+          <p>{t("retrocert-login.help-text")}</p>
+          <Form onSubmit={handleSubmit}>
+            <Row>
+              <Form.Group controlId="formLastName" className="col-md-6">
+                <Form.Label>{t("retrocert-login.last-name-label")}</Form.Label>
+                <Form.Control
+                  type="text"
+                  value={lastName}
+                  onChange={(e) => handleChange(e, setLastName)}
                 />
-          </Form.Group>
-
-          <Form.Group controlId="formEDDCAN">
-            <Form.Label>{t("retro-cert-auth-eddcan")}</Form.Label>
-            <Form.Control
-                type="text"
-                placeholder={t("retro-cert-auth-eddcan-placeholder")}
-                onChange={e => handleChange(e, setEDDCAN)}
+              </Form.Group>
+            </Row>
+            <Row>
+              <Form.Group controlId="formEDDCAN" className="col-md-6">
+                <Form.Label>{t("retrocert-login.eddcan-label")}</Form.Label>
+                <Form.Control
+                  type="text"
+                  value={eddcan}
+                  onChange={(e) => handleChange(e, setEDDCAN)}
                 />
-          </Form.Group>
-
-          <Form.Group controlId="formSSN">
-            <Form.Label>{t("retro-cert-auth-ssn")}</Form.Label>
-            <Form.Control
-                type="password"
-                placeholder={t("retro-cert-auth-ssn-placeholder")}
-                onChange={e => handleChange(e, setSSN)}
+              </Form.Group>
+            </Row>
+            <Row>
+              <Form.Group controlId="formSSN" className="col-md-6">
+                <Form.Label>{t("retrocert-login.ssn-label")}</Form.Label>
+                <Form.Control
+                  type="password"
+                  value={ssn}
+                  onChange={(e) => handleChange(e, setSSN)}
                 />
-          </Form.Group>
-          {/* reCAPTCHA v3 */}
-          <Button variant="secondary" type="submit">
-            {t("retro-cert-auth-find-me")}
-          </Button>
-        </Form>
-        </div>
-        <div className="container p-4">
-          {/* For testing, show the response JSON. */}
-          {userData && userData.status !== "not-logged-in" ? JSON.stringify(userData) : ""}
+              </Form.Group>
+            </Row>
+            <Row>
+              <Form.Group controlId="formReCaptcha" className="col-md-6">
+                <ReCAPTCHA
+                  sitekey="6Lf-DQEVAAAAABCMwJ-Gnbqec08RuiPhMZPtZPm9"
+                  // eslint-disable-next-line no-console
+                  onChange={(e) => console.log(e)}
+                />
+                <Form.Text className="text-muted">
+                  {t("retrocert-login.recaptcha-text")}
+                </Form.Text>
+              </Form.Group>
+            </Row>
+            <Row>
+              {userData && userData.status === "wrong-eddcan" && (
+                <Alert variant="danger">
```

Comment:
minor, you could move the Alert elements outside since they appear in all three of these